concent_api/core/tests/test_unit_subtask_helpers.py

```diff
@@ -63,20 +63,20 @@ def test_that_if_more_conditions_given_and_object_does_not_exist_function_return
         self.assertIsNone(subtask)
 
     def test_that_if_only_object_id_given_and_queryset_exist_function_returns_expected_object(self) -> None:
-        subtask = get_one_or_none(Subtask.objects.select_for_update(), subtask_id=self.task_to_compute.compute_task_def['subtask_id'])
+        subtask = get_one_or_none(Subtask.objects.all(), subtask_id=self.task_to_compute.compute_task_def['subtask_id'])
         self.assertEqual(self.subtask, subtask)
 
     def test_that_if_more_conditions_given_and_queryset_exist_function_returns_expected_object(self) -> None:
         subtask = get_one_or_none(
-            Subtask.objects.select_for_update(),
+            Subtask.objects.all(),
             subtask_id=self.task_to_compute.compute_task_def['subtask_id'],
             state=Subtask.SubtaskState.FORCING_REPORT.name,  # pylint: disable=no-member
         )
         self.assertEqual(self.subtask, subtask)
 
     def test_that_if_more_conditions_given_and_queryset_does_not_exist_function_returns_none(self) -> None:
         subtask = get_one_or_none(
-            Subtask.objects.select_for_update(),
+            Subtask.objects.all(),
```

Comment:
why is it changed? `get_one_or_none` should work with `select_for_update` as well. I think it should stay at least one test with `select_for_update`

Reply:
But this test doesn't check that, so it should work for any queryset. And here there was a problem that `select_for_update()` works only in transaction, so when I have switched to `TransactionTestCase` (which, opposite to its name, is not run in transaction) there was an exception here.